test/functional/TestUtilities/src/org/openj9/test/util/PlatformInfo.java

```diff
@@ -33,6 +35,16 @@ public static boolean isMacOS() {
 		return ((null != osName) && osName.startsWith("Mac OS"));
 	}
 
+	/**
+	 * Test if this is a pure OpenJ9 JDK, not a derivative or alternate VM.
+	 * This is used by tests which require specific tools or packages with the JVM.
+	 *
+	 * @return true if this is a stock OpenJ9 JDK
+	 */
+	public static boolean isOpenJ9() {
```

Comment:
I'm concerned this test is not going to be robust enough. While it may work as desired today, it may not give the desired results in the future, in particular if OpenJ9 is picked up and used in other ways.

If you want to know if jps and jstack are supported, how about looking for these files?

Reply:
I do in fact look for these tools elsewhere, but I wanted to check early and explicitly.

Reply:
Also, I want to check that the tool under test is present when it should be.